<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget_5">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>串口名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_serialName_3">
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="pushButton_scanSerial_3">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>扫描串口</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QRadioButton" name="radioButton_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_openSerial_3">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>打开串口</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_start_3">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>固有频率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_frequency_3">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>模式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_pattern_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>定电流模式</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>控制模式</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>定电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_current_3">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_22">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <item>
           <widget class="QLabel" name="label_23">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>频段电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_bandCurrent_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>较大电流</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>较小电流</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>操作日志：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_operationLogs_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_26">
          <item>
           <widget class="QPushButton" name="pushButton_frequencyDomain_2">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>频域分析</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_timeDoamin_2">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>时域分析</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>298</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCustomPlot" name="cusplot_3" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="horizontalScrollBar_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCustomPlot" name="cusplot_frequent" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMR_TMD">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuMR_TMD"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
